langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -43,6 +47,14 @@ public isolated function length(map<any|error> m) returns int =@java:Method {
 # The function `entries` can be used to iterate over the keys and members together.
 # The function `keys` can be used to iterator over just the keys.
 #
+# ```ballerina
+# map<int> marks = {"Carl": 85, "Bob": 50, "Max": 60};
+# object {
+#     public isolated function next() returns record {|int value;|}?;
+# } iterator = marks.iterator();
```

Comment:
```suggestion
# object {
#     public isolated function next() returns record {|int value;|}?;
# } iterator = {"Carl": 85, "Bob": 50, "Max": 60}.iterator();
```

Reply:
This code already has a bit more code than other examples. Doesn't this suggestion add extra complexity?

Reply:
Same as https://github.com/ballerina-platform/ballerina-lang/pull/39073/files#diff-2c948d23bf7d2faa264a43f44ca32e7f64f06b4dfb220764247218037b2d10f4R56, right? I don't see much of a difference.

Reply:
Ah okay. Let's add this suggestion.